packages/@ionic/cli/src/commands/capacitor/base.ts

```diff
@@ -236,6 +249,17 @@ export abstract class CapacitorCommand extends Command {
 
         manifest.enableCleartextTraffic();
         await manifest.save();
+      }      
+
+      if (platform === 'ios') {
```

Comment:
```suggestion
      if (platform === 'ios' && !options["external"]) {
```
when passing `--external` option it would use a local IP to load the app and ATS won't block the connection, so would be good to not edit the plist if it's not necessary